valio-master/Valio/valio.json

```diff
@@ -0,0 +1,196 @@
+[
+  {
+    "title": "Thursday",
+    "subtitle": "June 5th",
+    "items": [
+      {
+        "time": "8:00 am",
```

Comment:
Same idea as above, I would keep a timestamp for the meeting instead of a string in some hour format. People some times have different locales and e.g. have a 24h clock set in the iPhone settings